### PR TITLE
Update x-plat guide for RC1

### PR DESCRIPTION
**Pull request by natemcmaster**
Use CoreCLR by default in the tutorials.

### CODE REVIEW

**Comment by natemcmaster** [comment]
cc @rowanmiller

**Comment by natemcmaster** [comment]
I investigated showing VS Code in the guides, but this still has a dependency on Mono (see https://github.com/OmniSharp/omnisharp-roslyn/issues/336).

File: docs/getting-started/linux.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ This walkthrough will create a simple console application using ASP.NET 5 and
 the SQLite provider.
 
 .. note::
-    This article was written for beta 8 of ASP.NET and EF7.
+    This article was written for RC1 of ASP.NET and EF7.
 
     You can find nightly builds of the EF7 code base hosted on https://www.myget.org/gallery/aspnetvnext/ but the code base is rapidly changing and we do not maintain up-to-date documentation for getting started.
```
Comment (rowanmiller) [inline docs/getting-started/linux.rst]: Can you swap this out for `.. include:: /rc1-notice.txt` and put it as the first thing in the file (trying to standardize on that across all docs)

File: docs/getting-started/osx.rst [hunk 2]
```diff
@@ -5,7 +5,7 @@ This walkthrough will create a simple console application using ASP.NET 5 and
 the SQLite provider.
 
 .. note::
-    This article was written for OS X Mavericks or newer. It uses beta 8 of ASP.NET and EF7.
```
Comment (rowanmiller) [inline docs/getting-started/osx.rst]: Swap this out for `.. include:: /rc1-notice.txt` too

File: docs/getting-started/x-plat/sample/src/ConsoleApp/Model.cs [hunk 3]
```diff
@@ -13,9 +12,8 @@ public class BloggingContext : DbContext
 
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
-            var appEnv = CallContextServiceLocator.Locator.ServiceProvider
-                            .GetRequiredService<IApplicationEnvironment>();
-            optionsBuilder.UseSqlite($"Data Source={ appEnv.ApplicationBasePath }/blog.db");
+            var path = PlatformServices.Default.Application.ApplicationBasePath;
+            optionsBuilder.UseSqlite("Filename=" + Path.Combine(path, "blog.db"));
```
Comment (rowanmiller) [inline docs/getting-started/x-plat/sample/src/ConsoleApp/Model.cs]: Can this just be relative now too?

**Comment by rowanmiller** [comment]
:shipit:

File: docs/getting-started/x-plat/sample/src/ConsoleApp/Model.cs (hunk 3 above)
Comment (natemcmaster) [inline docs/getting-started/x-plat/sample/src/ConsoleApp/Model.cs]: Almost. RC1 only does this on UWP. Our next release will include the full fix.

File: docs/getting-started/osx.rst (hunk 2 above)
Comment (natemcmaster) [inline docs/getting-started/osx.rst]: Will do.

**Pull request merged**